classes.ipynb

```diff
@@ -0,0 +1,1597 @@
+    "            if(self.start_date == 'inception'):\n",
+    "                start_date = stockData.loc[stockData['Stock'] == stockChoices[i], 'Inception_Date'].iloc[0]\n",
+    "                print(start_date)\n",
+    "            else:\n",
+    "                start_date = self.start_date\n",
+    "            \n",
+    "            print(\"Stock: \", stockChoices[i])\n",
+    "\n",
+    "            stock_data = etf_dict[stockChoices[i]]\n",
+    "\n",
+    "            stock_exp_ratio = stockData.loc[stockData['Stock'] == stockChoices[i], 'Expense_Ratio'].iloc[0]\n",
+    "            print(\"Expense ratio: \", stock_exp_ratio)\n",
+    "            #investment_amount = input(\"Enter Investment amount: \\n\")\n",
+    "            investment_amount = 1000\n",
+    "            frequency = input(\"Enter frequency (daily, weekly, biweekly, monthly): \\n\")\n",
+    "            investment_day = input(\"Enter day of the week to invest: \\n\") \n",
+    "\n",
+    "            total_investment, final_value, annualized_return, annualized_roi, sharpe_ratio, longest_losing_streak = trade.dollar_cost_average(stock_data, investment_amount, frequency, investment_day, stock_exp_ratio, self.start_date, self.end_date)\n",
+    "\n",
+    "            dca_portfolio.loc[len(dca_portfolio.index)] = [stockChoices[i],stock_exp_ratio, investment_amount, frequency,investment_day, total_investment, final_value, annualized_return, annualized_roi, sharpe_ratio, longest_losing_streak ]\n",
+    "            print(f'Total investment:', total_investment)\n",
+    "            print(f'Final Value: ', final_value)\n",
+    "            print(f'Annualized return: {annualized_return:}')\n",
+    "            print(f'Annualized ROI: {annualized_roi:}')\n",
+    "            print(f'Sharpe ratio: {sharpe_ratio:.2}')\n",
+    "            print(f'Longest losing streak (weeks): {longest_losing_streak}\\n')\n",
+    "            \n",
+    "        \n",
+    "        \n",
+    "        return dca_portfolio\n",
+    "\n",
+    "\n",
+    "    def dca_portfolio_multi(self):\n",
+    "        \"\"\"\n",
+    "        This function is used to preform a dollar cost average strategy on a portfolio that contains more than one stock.\n",
+    "        \n",
+    "        Args:\n",
+    "            input_etfs : list\n",
+    "                A list that contains each of the stocks that will make up the portfolio.\n",
+    "            frequency_list : list\n",
+    "                A list that contains how frequently a stock will get traded. If there are 3 ETFs listed there needs to be three frequencies listed.\n",
+    "            days : list\n",
+    "                This list determines which day of the week the trading will occur.  If there are 3 ETFs listed there needs to be three frequencies listed.\n",
+    "            start_date : dateTime\n",
+    "                Determines which date the trading will start.\n",
+    "            end_date : dateTime\n",
```

Comment:
The function declaration states no input arguments is needed. But the docstring seems the other way. Please fix it and make them consistent